static_files/css/style.css

```diff
@@ -7,7 +7,34 @@ html {
 }
 
 @import url('https://fonts.googleapis.com/css2?family=Dongle:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Chenla&family=Dongle:wght@300&display=swap');
 
 body{
     font-family: "Dongle", sans-serif;
+    min-height: 100vh;
 }
+
+header{
+    min-height: 50px;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+}
+
+main{
+    flex: 1;
+}
+
+.Footer {
+    min-height: 50px;
+    font-family: "Nunito", sans-serif;;
+    text-align: center;
+    color: #CCCCCC;
+    font-size: small;
```

Comment:
Absolute value keyword is interesting from the point of view of responsiveness, but limiting because it has a set of possible values ​​and also may not have the necessary precision to keep the final application consistent with the prototype, you know? Here is an example of the difference between footer when using px and small
![image](https://user-images.githubusercontent.com/105747728/232608414-15d8be15-443c-48fd-b1b4-0f1dc6739cbd.png)
![image](https://user-images.githubusercontent.com/105747728/232608453-09db931f-56d2-48c9-b8dc-6cd0f2f29194.png)